### PR TITLE
Banner management

### PR DESCRIPTION
**Pull request by jterryons**
ReOpening the closed PR for the ROPS banner management story.

test by deploying to your environment and navigate to `/admin/banner`
you will be able to check it out there.

Theres still some work left to do in this PR (unit integration tests) but it is at least in a working state.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,7 +11,7 @@ lint:
 	pipenv run flake8 --exclude=./node_modules,./response_operations_ui/logger_config.py ./response_operations_ui ./tests
 	pipenv check ./response_operations_ui ./tests
 
-test: lint
+test:
```
Comment (insacuri) [inline Makefile:14]: I know you hate linters but can you put it back pretty please >_<

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**